### DISCUSSION TITLE
Mod compatibility list

### DISCUSSION THREAD

**Discussion by ChoiceTheorem** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#39] [Ideas]
Post the mod you want to have compatibility with CTOV here.
Existing compats:
-   [x] Built-in : Guard villager, Waystone, Bountiful, Worldgen
-   [x] [**Waystone (Before version 3.2.1)**](https://modrinth.com/datapack/ctov-waystone-compat) 
-   [x] [**Farmer's delight**](https://modrinth.com/datapack/ctov-farmers-delight-compat)
-   [x] [**More villagers**](https://modrinth.com/datapack/ctov-more-villagers-compat)
-   [x] [**Croptopia**](https://modrinth.com/datapack/ctov-croptopia-compat)
-   [x] [**Bountiful (Before version 3.2.1)**](https://modrinth.com/datapack/ctov-bountiful-compat)
-   [x] [**Friend and Foe**](https://modrinth.com/datapack/ctov-friends-and-foes-compat)
-   [x] [**Immersive Engineering**](https://modrinth.com/datapack/ctov-immersive-engineering-compat)
-   [x] [**Incubation**](https://modrinth.com/datapack/ctov-incubation-compat)
-   [x] [**Chef's delight**](https://modrinth.com/datapack/ctov-chefs-delight-compat)
-   [x] [**Beautify**](https://modrinth.com/datapack/ctov-beautify-compat)
-   [x] [**Vampirism**](https://modrinth.com/mod/vampirism-integrations)
-   [x] [**Domestication Innovation**](https://modrinth.com/datapack/ctov-domesticated-innovation-compat)
-   [x] [**Wizards**](https://modrinth.com/datapack/ctov-wizards-compat)
-   [x] [**Villager Plus**](https://modrinth.com/datapack/ctov-villagers-plus-compat)
-   [x] [**Advance peripheral**](https://modrinth.com/datapack/ctov-advanced-peripheral-compat)
-   [x] [**BYG**](https://modrinth.com/datapack/ctov-byg-compat)
-   [x] [**Savage and Ravage**](https://modrinth.com/datapack/ctov-savage-and-ravage-compat)
-   [x] [**Vending Machine**](https://modrinth.com/datapack/ctov-vending-machine-compat)
-   [x] [**All bark All bite**](https://modrinth.com/datapack/ctov-abab-compat)
-   [x] [**Gazebos**](https://modrinth.com/datapack/ctov-gazebo-compat)
-   [x] [**Paladins & Priests**](https://modrinth.com/datapack/ctov-paladins-n-priests-compat)
-   [x] [**Paragliders**](https://modrinth.com/datapack/ctov-paragliders-compat)
-   [x] Pneumatic craft
-   [x] Monobank
-   [x] Jelly Fishing
-   [x] Wares
-   [x] Iron's Spells 'n Spellbooks
-   [x] Create structures
-   [ ] Questager - Wait for the mod rewriting

**Comment by nsc234** [comment]
What about the guard villagers mod? do they naturally fit the villagers since they are not new biomes- just reworks?

**Reply by ChoiceTheorem** [reply]
Already have compatibility a long time ago. Built-in even

**Reply by nsc234** [reply]
Ight thank you cool mod

**Comment by ChoiceTheorem** [comment]
From #74 
mods i would like to get datapack addons

- [ ] [Lightmans Currency](https://www.curseforge.com/minecraft/mc-mods/lightmans-currency)
- [x] [Create structers](https://www.curseforge.com/minecraft/mc-mods/create-structures)
- [x] [Floralis](https://www.curseforge.com/minecraft/mc-mods/floralis)

**Comment by ChoiceTheorem** [comment]
- [x] [Rat](https://www.curseforge.com/minecraft/mc-mods/rats)

**Reply by ScoutTheSaneless** [reply]
i second this addition!

**Comment by ChoiceTheorem** [comment]
- [x] [Better Archeology](https://www.curseforge.com/minecraft/mc-mods/better-archeology)

**Reply by Zamanos** [reply]
Thank you for considering adding this to your mod, I would greatly appreciate this!

**Reply by ChoiceTheorem** [reply]
Doesn't seem to contain features to make compat with.

**Reply by Zamanos** [reply]
Does that mean that the archeologist will appear in your modded villages?

**Comment by AngellusMortis** [comment]
This is probably a larger than normal request, but [TerraFirmaCraft](https://www.curseforge.com/minecraft/mc-mods/terrafirmacraft).

At least the best that can be with how much the mod changes world gen. [Docs](https://terrafirmacraft.github.io/Documentation/1.20.x/).

**Reply by ChoiceTheorem** [reply]
Too large to make compat with.

**Comment by Min-vi** [comment]
I’d like compatibility with [Just More Cakes](https://www.curseforge.com/minecraft/mc-mods/just-more-cakes)

**Comment by CuriouseCat** [comment]
will there be compatibility with TerraFirmaCraft?

**Comment by ChoiceTheorem** [comment]
If you means two mod coexistence, already is!
If you means seamless integration, isn't possible.

**Comment by IMoud47** [comment]


**Reply by IMoud47** [reply]
Compatibility for Minecraft Comes Alive Reborn #90
Amazing!

**Comment by AnyPruft** [comment]
The Graveyard compatibility. Or a datapack would be nice.
Currently the structures overlap.

[IMG](https://imgur.com/a/Fzns6x5)

**Comment by Routhinator** [comment]
I would like to see a way to map  villages to biomes based on wood type, as I am dealing with a pack that has extra forest biomes from Terrestria, Nature's Spirit and similar where they add a biodiverse set of forests and there's no reason a village can't spawn in these forest variants. Maybe there's another mod that could help with this. Would also be nice to be able to simply have wood in a village type swapped for a different wood in the world for forests with non-standard wood types.

**Reply by ChoiceTheorem** [reply]
But I think that a new village pallete is better for such mods which take more times than other mods.

**Reply by Routhinator** [reply]
Could we have an option or method of just telling the mod to add "forest_village" types to a list of biomes as valid? In my eyes the wood type being swapped is a nice to have, but more importantly a village not spawning because the worldgen picked one forest type over another randomly is more problematic.

**Reply by Routhinator** [reply]
I found a way to do this with Structify - doesn't swap the wood out but at least I can make villages that have a close enough feel spawn in biomes they would otherwise not spawn in.

**Comment by VerinSenpai** [comment]
disregard, lithostiched strikes again.

**Comment by PoundGueux** [comment]
Is it compatible with Terralith/Tectonic?? Ik terralith adds villages of its own

**Comment by ChoiceTheorem** [comment]
Always has been compatible, I guess it is very hard to read the wiki before
asking.
